Makefile.doc

```diff
@@ -199,7 +202,7 @@ install-doc-printable:
 	$(INSTALLLIB) doc/stdlib/Library.pdf $(FULLDOCDIR)/pdf
 	$(INSTALLLIB) doc/stdlib/Library.ps $(FULLDOCDIR)/ps
 
-install-doc-sphinx-html:
+install-doc-sphinx:
```

Comment:
This one didn't get renamed to refman

Reply:
Indeed. I'm not happy with the current location where the manual is installed (see #8524) so until this is fixed, I prefer not to change the name of this target. (This is not urgent and I need to discuss this with @cpitclaudel before.)

Reply:
@Zimmi48 I am not sure I am much competent on the make changes.